src/PackagePublisher/Program.cs

```diff
@@ -24,7 +24,9 @@ public static class Program
 
         // Do NOT log the key
         private static string _apiKey = null;
-        private static int _maxRetryCount = 10;
+        private static int _maxRetryCount = 5;
+        private static int _packagePushTimeoutInSeconds = 90;
+        private static int _semaphoreTimeoutInSeconds = _maxRetryCount * _packagePushTimeoutInSeconds;
```

Comment:
Wouldn't we have to account for the number of packages in here? Since all the pushes queue up at the onset, you give it at most 7.5 minutes which is usually enough, but that's at most it takes for one package to publish, not the 89 other packages also waiting to be pushed.